libraries/common/src/main/java/androidx/media3/common/util/Util.java

```diff
@@ -530,6 +574,23 @@ public static <T> void nullSafeListToArray(List<T> list, T[] array) {
     list.toArray(array);
   }
 
+  /**
+   * Creates a new Integer array from a List of Integers.
+   *
+   * @param list The list of integers to convert.
+   * @return Created array of integers.
+   */
+  @UnstableApi
+  @SuppressWarnings("nullness:assignment")
+  public static int[] nullSafeIntegerListToArray(List<Integer> list) {
```

Comment:
Would it be better to use [Ints.toArray(Collection)](https://guava.dev/releases/19.0/api/docs/com/google/common/primitives/Ints.html#toArray(java.util.Collection))?

Reply:
@tianyif  Done. Please see d4f8b7f7ed1cae89d725dfbf4d74492d3808356a  Thanks